src/spec/tests/core/usingDropdown.js

```diff
@@ -79,3 +71,25 @@ describe("using dropdown: init plugin with nationalMode=false", function() {
   });
 
 });
+
+describe("using dropdown: disabled input", () => {
+  beforeEach(function() {
+    intlSetup();
+    input = $("<input disabled>").appendTo("body");
+    iti = window.intlTelInput(input[0]);
+  });
+
+  afterEach(function() {
+    intlTeardown();
+  });
+
+  it("prevents the user from opening the dropdown using the keyboard", function() {
+    expect(getSelectedFlagContainer().attr('tabindex')).toEqual('-1');
+    expect(getSelectedFlagContainer().attr('aria-disabled')).toEqual('true');
```

Comment:
FYI, I tried the following:

```js
getSelectedFlagContainer()[0].dispatchEvent(new CustomEvent('focus'));
triggerKeyOnFlagsContainerElement("ArrowDown");
expect(getListElement()).not.toBeVisible();
```
But this doesn't work because `dispatchEvent` doesn't check the `disabled` attribute:  https://github.com/testing-library/dom-testing-library/issues/92#issuecomment-530249041